internal/mq/msgstream/mq_factory.go

```diff
@@ -65,13 +84,34 @@ func (f *PmsFactory) NewMsgStream(ctx context.Context) (MsgStream, error) {
 
 // NewTtMsgStream is used to generate a new TtMsgstream object
 func (f *PmsFactory) NewTtMsgStream(ctx context.Context) (MsgStream, error) {
-	pulsarClient, err := puslarmqwrapper.NewClient(pulsar.ClientOptions{URL: f.PulsarAddress})
+	auth, err := f.getAuthentication()
+	if err != nil {
+		return nil, err
+	}
+	clientOpts := pulsar.ClientOptions{
+		URL:            f.PulsarAddress,
+		Authentication: auth,
+	}
+
+	pulsarClient, err := puslarmqwrapper.NewClient(f.PulsarTenant, f.PulsarNameSpace, clientOpts)
 	if err != nil {
 		return nil, err
 	}
 	return NewMqTtMsgStream(ctx, f.ReceiveBufSize, f.PulsarBufSize, pulsarClient, f.dispatcherFactory.NewUnmarshalDispatcher())
 }
 
+func (f *PmsFactory) getAuthentication() (pulsar.Authentication, error) {
+	auth, err := pulsar.NewAuthentication(f.PulsarAuthPlugin, f.PulsarAuthParams)
+
+	if err != nil {
+		log.Error("build authencation from config failed, please check it!",
+			zap.String("authPlugin", f.PulsarAuthPlugin),
+			zap.String("authParams", f.PulsarAuthParams))
```

Comment:
Maybe we need add `zap.Error(err)` to show the exact error message from pulsar here.